WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -236,6 +243,33 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
         present(alertController, animated: true)
     }
 
+    /// Product Confirmation Delete alert
+    ///
+    func presentProductConfirmationDeleteAlert(completion: @escaping (Bool) -> ()) {
```

Comment:
nit: just a suggestion to move this alert setup to like `UIAlertController+Helpers` or some other helper class since `ProductFormViewController` is already pretty huge?

Reply:
nit: maybe the `Bool` parameter can have an internal description like:
```suggestion
    func presentProductConfirmationDeleteAlert(completion: @escaping (_ isConfirmed: Bool) -> ()) {
```

Reply:
not sure if it makes sense to move the alert logic to a helper since `ProductFormViewController` is already 1200 LOC? 🙂